src/components/CreateRules/UploadRulesSummary.vue

```diff
@@ -5,7 +5,7 @@
       <label class="upload-rules-title pl-2"><strong>Rules</strong></label>
     </div>
 
-    <div v-if="!getCreateRulesStep.valid" class="invalid-section">
+    <div v-if="!getCreateRulesStep.validationDetail.valid" class="invalid-section">
```

Comment:
Are you sure that `validationDetail` is not null? It's initialized to null in Filing Template Mixin so this line of code might error.

Same on line 20 below.

Reply:
OK, I see you initialize this in State Model. Looks like it's OK.